app/main.js

```diff
@@ -67,9 +71,9 @@ function createTrayIcon () {
   } else {
     appIcon = new Tray(iconFolder + '/stretchly_18x18.png')
   }
-  appIcon.setToolTip('stretchly - break time reminder app')
   isPaused = false
   appIcon.setContextMenu(getTrayMenu())
+  toolTipUpdater = setInterval(updateToolTip, 1000)
```

Comment:
I'm thinking maybe change the logic to be update every 10 secs, round to the nearest 5 seconds when under a minute.